src/SUMMARY.md

```diff
@@ -17,6 +17,7 @@
     - [Incremental compilation](./incremental-compilation.md)
     - [Debugging and Testing](./incrcomp-debugging.md)
 - [The parser](./the-parser.md)
+- [Testing](./testing.md)
```

Comment:
Nit: Testing is very vague. Maybe we should call this "How `#[test]` is implemented" or something like that.